### PR TITLE
Query for getting complete account details

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
Added `GetAccountDetails` query that returns allowances for **internal use only**. 
This returns all info of an account including allowance details. This query **WILL NOT** be documented in HAPI

### CODE REVIEW

**Review by superboo — APPROVED** [review]
LGTM

File: services/get_account_details.proto [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        /**
+         * The current balance of account in tinybars
+         */
+        uint64 balance = 8;
+
+        /**
+         * [Deprecated]. The threshold amount, in tinybars, at which a record is created of any
+         * transaction that decreases the balance of this account by more than the threshold
+         */
+        uint64 generate_send_record_threshold = 9 [deprecated=true];
```
Comment (tinker-michaelj) [inline services/get_account_details.proto:110]: We don't even have these thresholds in state any more; so probably no reason include them in this message. 😁

File: services/get_account_details.proto [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+         * insufficient funds, then it extends as long as possible. If it is empty when it expires,
+         * then it is deleted.
+         */
+        Duration auto_renew_period = 13;
+
+        /**
+         * All of the livehashes attached to the account (each of which is a hash along with the
+         * keys that authorized it and can delete it)
+         */
+        repeated LiveHash live_hashes = 14;
```
Comment (tinker-michaelj) [inline services/get_account_details.proto:139]: Also wouldn't include this.

**Review by tinker-michaelj — COMMENTED** [review]
Noticed a couple unsupported fields that we probably gain nothing from including in the response.

Also, we need a new `NetworkService` `rpc` [here](https://github.com/hashgraph/hedera-protobufs/blob/main/services/network_service.proto#L41); and a new `HederaFunctionality` constant [here](https://github.com/hashgraph/hedera-protobufs/blob/main/services/basic_types.proto#L1099).

File: services/get_account_details.proto (hunk 1 above)
Comment (Neeharika-Sompalli) [reply services/get_account_details.proto]: Removed them . Thanks Michael !

File: services/get_account_details.proto (hunk 2 above)
Comment (Neeharika-Sompalli) [reply services/get_account_details.proto]: Removed . Thanks Michael !

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Pull request merged**